components/merkledb/src/proof_list_index/proof.rs

```diff
@@ -521,6 +521,31 @@ impl<V: BinaryValue> ListProof<V> {
                 }
             })
     }
+
+    /// Provides access to the proof part of the view. Used in serialization.
+    pub fn proof_unchecked(&self) -> Vec<(ProofListKey, Hash)> {
```

Comment:
Same problem here as for `MapProof`s (#1477): because `ProtobufConvert` is not extracted as a separate crate, it's necessary to expose internal structure of MerkleDB. This doesn't look like a good decision in long term.